extra/1-currency-conversion.js

```diff
@@ -16,6 +19,11 @@ function convertToUSD() {}
 */
 
 function convertToBRL() {}
+function convertToUSD(value){
```

Comment:
You've got three functions with the same name here, this will cause a syntax error